lib/perl/Genome/Model/Command/Admin/ModelSummary.pm

```diff
@@ -202,6 +184,21 @@ sub execute {
     return 1;
 }
 
+sub _build_and_status_for_model {
```

Comment:
If genome/UR#85 were merged, this would look less like both of those, as we could use `-limit` in a `get()` to just get the one we need.  `current_build` would still need an iterator.

It might make sense to change `Genome::Model->latest_build` to accept an additional filter.